RadixWallet/Features/SecurityCenterFeature/SecurityFactors/FactorSourcesList.swift

```diff
@@ -0,0 +1,269 @@
+			case displayMnemonic(DisplayMnemonic.State)
+			case enterMnemonic(ImportMnemonicsFlowCoordinator.State)
+			case addMnemonic(ImportMnemonic.State)
+		}
+
+		@CasePathable
+		enum Action: Sendable, Equatable {
+			case detail(FactorSourceDetail.Action)
+			case displayMnemonic(DisplayMnemonic.Action)
+			case enterMnemonic(ImportMnemonicsFlowCoordinator.Action)
+			case addMnemonic(ImportMnemonic.Action)
+		}
+
+		var body: some ReducerOf<Self> {
+			Scope(state: \.detail, action: \.detail) {
+				FactorSourceDetail()
+			}
+			Scope(state: \.displayMnemonic, action: \.displayMnemonic) {
+				DisplayMnemonic()
+			}
+			Scope(state: \.enterMnemonic, action: \.enterMnemonic) {
+				ImportMnemonicsFlowCoordinator()
+			}
+			Scope(state: \.addMnemonic, action: \.addMnemonic) {
+				ImportMnemonic()
+			}
+		}
+	}
+
+	@Dependency(\.securityCenterClient) var securityCenterClient
+	@Dependency(\.factorSourcesClient) var factorSourcesClient
+	@Dependency(\.errorQueue) var errorQueue
+
+	var body: some ReducerOf<Self> {
+		Reduce(core)
+			.ifLet(destinationPath, action: /Action.destination) {
+				Destination()
+			}
+	}
+
+	private let destinationPath: WritableKeyPath<State, PresentationState<Destination.State>> = \.$destination
+
+	func reduce(into state: inout State, viewAction: ViewAction) -> Effect<Action> {
+		switch viewAction {
+		case .task:
+			return securityProblemsEffect()
+				.merge(with: entitiesEffect(state: state))
+
+		case let .rowTapped(row):
+			state.destination = .detail(.init(integrity: row.integrity))
+			return .none
+
+		case let .rowMessageTapped(row):
+			switch row.status {
+			case .seedPhraseWrittenDown, .notBackedUp:
+				return .none
+
+			case .seedPhraseNotRecoverable:
+				if let factorSourceId = row.integrity.factorSourceIdOfMnemonicToExport {
+					return exportMnemonic(factorSourceID: factorSourceId) {
+						state.destination = .displayMnemonic(.export($0, title: L10n.RevealSeedPhrase.title, context: .fromSettings))
+					}
+				} else {
+					return .none
+				}
+
+			case .lostFactorSource:
+				state.destination = .enterMnemonic(.init())
+				return .none
+			}
+
+		case .addButtonTapped:
+			switch state.kind {
+			case .device:
+				state.destination = .addMnemonic(
+					.init(
+						showCloseButton: false,
+						isWordCountFixed: true,
+						persistStrategy: .init(
+							factorSourceKindOfMnemonic: .babylon(isMain: false),
+							location: .intoKeychainAndProfile,
+							onMnemonicExistsStrategy: .appendWithCryptoParamaters
+						),
+						wordCount: .twentyFour
+					)
+				)
+			case .ledgerHqHardwareWallet, .offDeviceMnemonic, .arculusCard, .password:
+				// NOTE: Added `.device` support as placeholder, but not adding the logic for ledger (which we already support)
+				// since Matt mentioned we will probably always present this screen: https://zpl.io/wyqB6Bd
+				// and I don't want to add all the logic for checking if there is a CE or not just to migrate it later.
```

Comment:
Not clear why you cannot add the current .ledgerHqHardwareWallet flow

Reply:
Matt mentioned on latest design session that adding the `Enter Seed Phrase` as a placeholder was fine for devices, but that the Add Factor Source flow would be defined into something generic (as presenting the view I linked regardless of the factor source list you are viewing).

The Add Ledger Flow consist of 3 views: Add Ledger view, No P2P Alert & Add P2P view.

I didn't want to add all of these to this view if they are going to be removed later.